### ISSUE TITLE
More function under vehicles for players returns "No Vehicles".

### ISSUE THREAD

**Issue by Budji** [Cyberbyte-Studios/CyberWorks#84]
Hey,

We're having an issue with the more function. When searching a member out under "players" and going into the vehicles tab, the "More function" returns:
No vehicles.

Are there any fixes for this? It's getting quite annoying when you have 5000 cars and you're trying to find one person car. 

Regards,
Mathias

**Comment by MightySCollins** [comment]
Can you search with the vehicle page? The link might just be broken

**Comment by Budji** [comment]
When searching through the vehicles page, it still gets "No vehicles".

**Comment by MightySCollins** [comment]
Ok I will see what I can do... I have not looked at this in a while. Can you send me an email to sam@cyberbyte.org.uk it really help if you could provide me a sample of your database so I can run it with valid data.

**Comment by Budji** [comment]
I'll see what I can do, need a chat with our project lead before I can get a copy of the database - I'll pass you an email either way, just to let you know when I have an answer :-)

**Comment by MightySCollins** [comment]
Yep it does not actually need to be any data. Just one row or so which could be made up.
I just need to see the current structure. If you have custom tables then you can keep that as well.

**Comment by Budji** [comment]
I've send you an email now 👍

**Comment by aadriii87** [comment]
same issue here

**Comment by MightySCollins** [comment]
@aadriii87 if you can drop me an email I might have a fix for you.

**Comment by MightySCollins** [comment]
I will be on our discord server tonight to try and fix a few problems join https://discord.gg/wfFSFhX and message me if you want to help.

**Comment by gigsta12** [comment]
I also am dealing with the same issue

**Comment by aadriii87** [comment]
Was a fix created? Still issues.
